README.md

```diff
@@ -28,9 +28,9 @@ Please see the content in the [wiki](https://github.com/Azure/terraform-azurerm-
 ## Example
 
 ```terraform
-module "alz_landing_zone" {
+module "lz_vending" {
   source  = "Azure/lz-vending/azurerm"
-  version = "~>0.1.0"
+  version = "<version>" # change this to your desired version
```

Comment:
I get why you're doing this from a code maintenance perspective, but does this make it harder for consumers? I don't find this a big overhead to update on each release.

Reply:
Added a link to the hashicorp docs page, but left as placeholder